js/models/conversations.js

```diff
@@ -603,6 +603,7 @@
     hasSentFriendRequest() {
       const status = this.get('friendRequestStatus');
       return (
+        status === FriendRequestStatusEnum.pendingSend ||
```

Comment:
would adding this break any other logic?

Reply:
Nah it is only used in `onFriendRequestAccepted`, `onFriendRequestReceived`, and `notify` and I think they all make sense, if a friend request is in the process of being sent and you receive a friend request or acceptance to yours then you can assume the exchange is complete